src/nvim/marktree.c

```diff
@@ -1626,7 +1626,12 @@ static int damage_cmp(const void *s1, const void *s2)
 {
   Damage *d1 = (Damage *)s1, *d2 = (Damage *)s2;
   assert(d1->id != d2->id);
-  return d1->id > d2->id;
+  if (d1->id < d2->id) {
```

Comment:
given the assert above, `return d1->id > d2->id ? 1 : -1` would be enough.

Reply:
Amen